azure_backend_boilerplate/locals.tf

```diff
@@ -0,0 +1,90 @@
+locals {
+  application_insights_env_variables = var.enable_application_insights == false ? null : {
+    "APPINSIGHTS_INSTRUMENTATIONKEY"                  = azurerm_application_insights.this[0].instrumentation_key
+    "APPLICATIONINSIGHTS_CONNECTION_STRING"           = azurerm_application_insights.this[0].connection_string
+    "APPINSIGHTS_PROFILERFEATURE_VERSION"             = "1.0.0"
+    "APPINSIGHTS_SNAPSHOTFEATURE_VERSION"             = "1.0.0"
+    "ApplicationInsightsAgent_EXTENSION_VERSION"      = "~3"
+    "XDT_MicrosoftApplicationInsights_BaseExtensions" = "disabled"
+    "DiagnosticServices_EXTENSION_VERSION"            = "~3"
+    "InstrumentationEngine_EXTENSION_VERSION"         = "disabled"
+    "SnapshotDebugger_EXTENSION_VERSION"              = "disabled"
+    "XDT_MicrosoftApplicationInsights_Mode"           = "recommended"
+    "XDT_MicrosoftApplicationInsights_PreemptSdk"     = "disabled"
+    "APPLICATIONINSIGHTS_CONFIGURATION_CONTENT"       = var.application_insights_CONFIGURATION_CONTENT
+  }
+
+  subnet_app = [
+    {
+      name             = "snet-${var.project}-${var.environment}-app"
+      address_prefixes = ["10.0.255.192/27"]
```

Comment:
Shall we move those addresses to variables, especially there is already variable for vnet address space?
```hcl
resource "azurerm_virtual_network" "this" {
  name                = "vnet-${var.project}-${var.environment}"
  location            = azurerm_resource_group.this.location
  resource_group_name = azurerm_resource_group.this.name
  address_space       = var.vnet_address_space
}
```

Reply:
That makes sense, however, we don't think either of these values will be changed. I will change that in a second.